.travis.yml

```diff
@@ -6,11 +6,9 @@ script: "bundle exec rake validate && bundle exec rake lint && bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.0.0
```

Comment:
Sigh. I don't want to support 1.8.7 any more. Ruby is EOL, puppet versions that only support that are eol, and PE versions using that are EOL. Are you saying I just need to delete it from the readme too? :)